author/backends.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-#!/usr/bin/env python
+# !/usr/bin/env python
```

Comment:
Actually, its because the shebang should be first line before the coding.
https://intellij-support.jetbrains.com/hc/en-us/community/posts/206601705--Reformat-Code-breaks-Shebang-line